libexec/basher-install

```diff
@@ -61,6 +61,9 @@ fi
 
 if [[ "$package" = */*@* ]]; then
   IFS=@ read -r package ref <<< "$package"
+  if [ "$custom_folder" = "false" ]; then
```

Comment:
L38 nit

"argunment" -> "argument"

Not really related to this PR, I noticed it and wanted to at least write it down